gammapy/background/off_data_background_maker.py

```diff
@@ -0,0 +1,248 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import os
+import logging
+import numpy as np
+from astropy.table import Table
+import astropy.units as u
+from astropy.table import join as table_join
+from ..data import ObservationTable, ObservationGroupAxis, ObservationGroups
+from .models import CubeBackgroundModel
+from .models import EnergyOffsetBackgroundModel
+from ..utils.energy import EnergyBounds
+from ..utils.axis import sqrt_space
+from ..extern.pathlib import Path
+
+__all__ = [
+    'OffDataBackgroundMaker',
+]
+
+log = logging.getLogger(__name__)
+
+
+class OffDataBackgroundMaker(object):
+    def __init__(self, data_store, outdir=None, run_list=None, obs_table_grouped_filename=None,
+                 group_table_filename=None):
+        """OffDataBackgroundMaker class.
```

Comment:
We always put the docstring for a class at the class level, and no docstring for `__init__`.
I.e. put this docstring here:
```
class MyClass(object):
    """Docstring here.
    """"
    def __init__():
         # no docstring here
```